src/main/java/com/vaadin/demo/component/confirmdialog/ConfirmDialogConfirmButton.java

```diff
@@ -0,0 +1,52 @@
+package com.vaadin.demo.component.confirmdialog;
+
+import com.vaadin.flow.component.Html;
+import com.vaadin.flow.component.button.Button;
+import com.vaadin.flow.component.confirmdialog.ConfirmDialog;
+import com.vaadin.flow.component.html.Div;
+import com.vaadin.flow.component.html.Span;
+import com.vaadin.flow.component.orderedlayout.FlexComponent;
+import com.vaadin.flow.component.orderedlayout.HorizontalLayout;
+import com.vaadin.flow.router.Route;
+import com.vaadin.demo.DemoExporter; // hidden-full-source-line
+
+@Route("confirm-dialog-confirm-button")
+public class ConfirmDialogConfirmButton extends Div {
+
+  private Span status;
+
+  public ConfirmDialogConfirmButton() {
+    HorizontalLayout layout = new HorizontalLayout();
+    layout.setAlignItems(FlexComponent.Alignment.CENTER);
+    layout.setJustifyContentMode(FlexComponent.JustifyContentMode.CENTER);
+
+    // tag::snippet[]
+    status = new Span();
+    status.setVisible(false);
+
+    ConfirmDialog dialog = new ConfirmDialog();
+    dialog.setHeader("Export failed");
+    dialog.setText(
+      new Html("An error occurred while exporting <b>Report Q4</b>. Please try again. If the problem persists, please contact <a href=\"mailto:support@company.com\">support@company.com</a>.")
```

Comment:
An exception is thrown at this line because of having both the `<a>` and `<b>` nodes at the parent level. Wrap the entire content in a `<span>` to fix this. Replacing the `Html` object with a `Span` object won't work because these tags will be escaped.
```suggestion
      new Html("<span>An error occurred while exporting <b>Report Q4</b>. Please try again. If the problem persists, please contact <a href=\"mailto:support@company.com\">support@company.com</a>.</span>")
```

Reply:
Ah, good catch! Changed to paragraph.